trl/trainer/reward_trainer.py

```diff
@@ -28,7 +28,7 @@
 
 
 if is_peft_available():
-    from peft import PeftModel, get_peft_model
+    from peft import PeftModel, get_peft_model, prepare_model_for_int8_training
```

Comment:
what about int4?

Reply:
It works for both, currently in PEFT there is a method  called `prepare_model_for_kbit_training`, but we did not made a release yet (they're the same method, just changed the name).